platform/android/dependencies.gradle

```diff
@@ -7,21 +7,21 @@ ext {
     versionCode = 11
     versionName = "5.0.0"
 
-    supportLibVersion = "25.3.1"
-    leakCanaryVersion = '1.5'
+    mapboxServicesVersion = "2.2.0-SNAPSHOT"
+    supportLibVersion = "25.1.0"
```

Comment:
this version matches the one found in MAS atm.
the supportLib from LOST was excluded.